src/modules/taskClass.js

```diff
@@ -0,0 +1,50 @@
+export default class TodoList {
+  allTodos = [];
+
+  saveTodo() {
+    const todos = JSON.stringify(this.allTodos);
+    localStorage.setItem('todos', todos);
+  }
+
+  addTodo(description) {
+    const todo = {
+      description,
+      completed: false,
+      index: this.allTodos.length + 1,
+    };
+    this.allTodos.push(todo);
+    this.saveTodo();
+  }
+
+  deleteTodo(indx) {
+    this.allTodos.splice(indx - 1, 1);
+    this.allTodos.forEach((t) => {
+      if (t.index > indx) {
+        t.index -= 1;
+      }
+    });
+    this.saveTodo();
+  }
+
+  getStoredTodos() {
+    this.allTodos = JSON.parse(localStorage.getItem('todos'));
+  }
+
+  editTodo(newValue, index) {
+    this.allTodos[index - 1].description = newValue;
+    this.saveTodo();
+  }
```

Comment:
- [ ] When I edit a task and give a space I got a bug when I reload the page. 

![Capture](https://user-images.githubusercontent.com/64173189/154594033-89b96b9b-a5f9-40a0-91b7-ba1d17160cce.JPG)

Kindly fix this minor issue. 😄 💪

Reply:
Resolved. Thanks